submissions/AntonKottans/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
```

Comment:
According to the HTML specification - all single tags should be without slashes "/". Could you please remove all these slashes?
 https://codebra.ru/en/lessons-html/dating/2/1